tests/Api.Tests/Endpoints/AuthenticatorsEndpointsTests.cs

```diff
@@ -0,0 +1,201 @@
+using System.Collections.Immutable;
+using AutoFixture;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Passwordless.Api.Endpoints;
+using Passwordless.Common.Models.Authenticators;
+using Passwordless.Service;
+using Passwordless.Service.Features;
+using Passwordless.Service.Helpers;
+
+namespace Passwordless.Api.Tests.Endpoints;
+
+public class AuthenticatorsEndpointsTests
+{
+    private readonly Fixture _fixture = new();
+
+    #region ListConfiguredAuthenticatorsAsync
+    [Fact]
+    public async Task ListConfiguredAuthenticatorsAsync_Throws_Forbidden_WhenAttestationIsNotAllowed()
```

Comment:
Can we perform these tests at an integration level instead?

Reply:
@Tyrrrz Yeah, but I wanted to refactor the integration tests first, so less steps would be required to create an app, etc.

Reply:
@Tyrrrz  done